drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -366,14 +369,21 @@ def can_see_client_details?(user)
             client["#{variant_name}_#{parts[:key]}"] = value
             next unless value
 
+            # Use the previously calculated household_type, for now, just get the first for the client
+            # that matches one of the prior calculations
+            project_client = project_clients[client_id].detect do |pc|
+              pc[:for_question].in?(parts[:household_type_keys])
+            end
+
             # These are only system level
-            project_clients << {
+            project_clients[client_id] ||= Set.new
+            project_clients[client_id] << {
               report_id: id,
               client_id: client_id,
               project_id: nil,
               for_question: parts[:key], # allows limiting for a specific response
               period: variant_name,
-              household_type: nil,
+              household_type: project_client[:household_type],
```

Comment:
Same question --  either 379 will never fire, or I think this needs to be
```suggestion
              household_type: project_client.try(:[], :household_type),
```
?